src/pal/pal_windows.h

```diff
@@ -127,5 +196,8 @@ namespace snmalloc
 #  endif
     }
   };
+  HEADER_GLOBAL std::atomic<uint64_t> PALWindows::pressure_epoch;
```

Comment:
What does the code gen do for this?  Hopefully doesn't do thread safe initialisation.